code/src/main.tsx

```diff
@@ -1,19 +1,25 @@
 import { createRoot } from "react-dom/client";
-import { StyledEngineProvider, ThemeProvider } from "@mui/material/styles";
+import { Provider } from "react-redux";
 import I18nProivider from "@/shared/i18n/I18nProvider";
+import { StyledEngineProvider, ThemeProvider } from "@mui/material/styles";
```

Comment:
Please double check if we need to import these components from `@mui/material/styles`. It seems like approach from previous mui versions, maybe we can use just `@mui/material`

Reply:
fixed